starter-code/src/viking.js

```diff
@@ -1,11 +1,48 @@
 // Soldier
-class Soldier {}
+class Soldier {
+    constructor(health, strength) {
+        this.health = health;
+        this.strength = strength;
+        Soldier.prototype.attack = function attack() {return this.strength}
+        Soldier.prototype.receiveDamage = function(damage) {this.health -= damage}
```

Comment:
Using the class syntax, this isn't how you are supposed to declare the class methods, the constructor is for the class properties, and then you declare it's methods within the class as follows : 

```
class Soldier {
  constructor(health, strength) {
    this.health = health;
    this.strength = strength;
  }

  attack() {
    return this.strength;
  }
  receiveDamage(damage) {
    this.health -= damage;
  }
}
```